nixos/modules/services/libeufin/common.nix

```diff
@@ -0,0 +1,161 @@
+# TODO: create a common module generator for Taler and Libeufin?
+libeufinComponent:
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+{
+  options.services.libeufin.${libeufinComponent} = {
+    enable = lib.mkEnableOption "libeufin core banking system and web interface";
+    package = lib.mkPackageOption pkgs "libeufin" { };
+    debug = lib.mkEnableOption "debug logging";
+    # TODO: find a better way to
+    openFirewall = lib.mkEnableOption "Open ports in the firewall";
```

Comment:
This is an incomprehensible comment left over from my review patch. What I meant is that the resulting doccomment for `openFirewall` is `Whether to Open ports in the firewall.`. I was thinking maybe it should be worded better. At the very least the first letter should be lowercased.